azure-pipelines.yml

```diff
@@ -0,0 +1,24 @@
+steps:
+- powershell: |
+   Import-Module .\tools\appveyor.psm1; Invoke-AppveyorInstall
+   if ($IsCoreClr) {
+     Invoke-AppveyorBuild -CheckoutPath $env:BUILD_SOURCESDIRECTORY -BuildConfiguration Release -BuildType 'NetStandard'
+   }
+   else {
+     Invoke-AppveyorBuild -CheckoutPath $env:BUILD_SOURCESDIRECTORY -BuildConfiguration Release -BuildType 'FullCLR'
+   }
+   $modulePath = $env:PSModulePath.Split([System.IO.Path]::PathSeparator) | Where-Object { Test-Path $_} | Select-Object -First 1
+   Copy-Item "$env:BUILD_SOURCESDIRECTORY\out\PSScriptAnalyzer" "$modulePath\" -Recurse -Force
+   $testScripts = "$env:BUILD_SOURCESDIRECTORY\Tests\Engine","$env:BUILD_SOURCESDIRECTORY\Tests\Rules"
+   $testResults = Invoke-Pester -Script $testScripts -OutputFormat NUnitXml -OutputFile "TestResults.xml" -PassThru 
+  displayName: 'Install, Build and Test'
+- task: PublishTestResults@2
+  displayName: 'Publish Test Results TestResults.xml'
+  inputs:
+    testRunner: NUnit
+    testResultsFiles: TestResults.xml
+- task: PublishBuildArtifacts@1
+  displayName: 'Publish Artifact: PSScriptAnalyzer'
```

Comment:
This currently won't work in an PR (I have an issue open to add this feature.)

See an example here how to skip this:  https://github.com/PowerShell/PowerShell/blob/f90fa8afc1d96603811f67fe95a6c3fbc74497c0/.vsts-ci/mac.yml#L60

Reply:
Ok, thanks, I did not know that. Is there a tracking issue in https://github.com/Microsoft/vsts-tasks ?